modules/markup/markdown/markdown_test.go

```diff
@@ -634,6 +638,8 @@ space</p>
 <a href="/wiki/file.bin" rel="nofollow">local link</a><br/>
 <a href="https://example.com" rel="nofollow">remote link</a><br/>
 <a href="/wiki/raw/image.jpg" target="_blank" rel="nofollow noopener"><img src="/wiki/raw/image.jpg" alt="local image"/></a><br/>
+<a href="/wiki/raw/attachments/cc42dfde" target="_blank" rel="nofollow noopener"><img src="/wiki/raw/attachments/cc42dfde" alt="local image"/></a><br/>
+<a href="/wiki/raw/attachments/cc42dfde" target="_blank" rel="nofollow noopener"><img src="/wiki/raw/attachments/cc42dfde" alt="local image"/></a><br/>
```

Comment:
Wait, you can upload attachments to the wiki?
Does this functionality really exist?

Reply:
No, none of these paths (must) exist. It's just a formatting test. `/delvh` and `delvh` are valid tests too if you like them more.

Reply:
Okay, I think I understand what you mean.
Agreed, it is not necessarily the case.
Nevertheless, the test makes it look like this URL most likely exists (instead of "might potentially exist").
But okay, fine.